src/ansys/mapdl/core/mapdl_console.py

```diff
@@ -284,8 +285,8 @@ def exit(self, close_log=True, timeout=3):
             try:
                 self._process.sendline("FINISH")
                 self._process.sendline("EXIT")
-            except:
-                pass
+            except Exception as e:
+                LOG.warning(f"Unable to exit ANSYS: {e}")
```

Comment:
```suggestion
                LOG.warning(f"Unable to exit ANSYS MAPDL: {e}")
```
Might be best to specify ANSYS MAPDL?